README.md

```
@@ -54,6 +54,22 @@ The STT/ASR backend service can be either [OpenAI API](https://platform.openai.c
      Model:
      Language Code:           multi
      ```
+   - Voxtral (Mistral):
+     ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)